### DISCUSSION TITLE
How are Text objects affacted by ttkbootstrap Style

### DISCUSSION THREAD

**Discussion by Yuankun123** [israel-dryer/ttkbootstrap#280] [Q&A] [answered]
I used to think that ttkbootstrap style does nothing to Text object. However, today I found that Text objects are affected. They become more beautiful but it seems that I can no longer change the font in it. Is it a bug?
Thank you!

**Comment by israel-dryer** [comment, marked as answer]
This is not a bug. I'm treating the Text widget like a standard ttk widget with a default style applied. However, this standard font style is overriding your font choice. I've made an update that should improve this in a PR #282 

With legacy tkinter widgets, you can stop the widget from getting automatic style updates by adding the argument `autostyle=False` when you create the widget. This will cause the widget to behave as it does normally. However, I realize that the font is something that is expected to be changed, so I've set the default font using the tkinter option database instead of with a style update; so this should give you the expected behavior.

I've pushed the update to Github, but probably won't make another release until next week. If you need this option now, you can install directly using `python -m pip install git+https://github.com/israel-dryer/ttkbootstrap` or you use the `autostyle=False` parameter and set the style yourself.

**Reply by Yuankun123** [reply]
Thank you very much!
